webapp/components/features/FollowList/FollowList.spec.js

```diff
@@ -0,0 +1,150 @@
+        return {}
+      },
+      'auth/isModerator': () => false,
+    }
+  })
+
+  describe('mount', () => {
+    beforeAll(() => {
+      store = new Vuex.Store({
+        getters,
+      })
+    })
+
+    describe('given a type', () => {
+      describe('of `following`', () => {
+        it('uses the `following` data on :user', () => {
+          const wrapper = Wrapper({ user, type: 'following' })
+          wrapper.find('.base-button').trigger('click')
+
+          expect(wrapper.vm.connections).toEqual(user.following)
+          expect(wrapper.vm.allConnectionsCount).toBe(user.followingCount)
+          expect(wrapper.emitted('fetchAllConnections')).toEqual([['following']])
+        })
+      })
+
+      describe('of `followedBy`', () => {
+        it('uses the `followedBy` data on :user', () => {
+          const wrapper = Wrapper({ type: 'followedBy' })
+          wrapper.find('.base-button').trigger('click')
+
+          expect(wrapper.vm.connections).toEqual(user.followedBy)
+          expect(wrapper.vm.allConnectionsCount).toBe(user.followedByCount)
+          expect(wrapper.emitted('fetchAllConnections')).toEqual([['followedBy']])
+        })
+      })
+    })
+
+    describe('given no type', () => {
+      it('defaults type to `following`', () => {
+        const wrapper = Wrapper()
+        expect(wrapper.vm.type).toBe('following')
+      })
+    })
+
+    describe('given a user', () => {
+      describe('without connections', () => {
+        it('displays the followingNobody message', () => {
+          const wrapper = Wrapper({ user: noConnectionsUser })
+          expect(wrapper.find('.nobody-message').text()).toBe(
+            `${noConnectionsUser.name} ${wrapper.vm.$t(`profile.network.followingNobody`)}`,
+          )
+        })
+
+        it('displays the followedByNobody message', () => {
+          const wrapper = Wrapper({ user: noConnectionsUser, type: 'followedBy' })
+          expect(wrapper.find('.nobody-message').text()).toBe(
+            `${noConnectionsUser.name} ${wrapper.vm.$t(`profile.network.followedByNobody`)}`,
+          )
+        })
+      })
+
+      describe('with up to 7 loaded connections', () => {
+        let wrapper
+        beforeAll(() => {
+          wrapper = Wrapper()
+        })
+
+        it(`renders the connections`, () => {
+          expect(wrapper.findAll('.user-teaser').length).toEqual(user.following.length)
+        })
+
+        it(`has a button to load all remaining connections`, async () => {
+          wrapper.find('.base-button').trigger('click')
+          expect(wrapper.emitted('fetchAllConnections')).toBeTruthy()
+        })
+      })
+
+      describe('with more than 7 loaded connections', () => {
+        let wrapper
+        beforeAll(() => {
+          wrapper = Wrapper({ user: allConnectionsUser })
+        })
+
+        it('renders the connections', () => {
+          expect(wrapper.findAll('.user-teaser')).toHaveLength(allConnectionsUser.followingCount)
+        })
+
```

Comment:
Here is what I expected before. Looks good. But how does the `.--overflow` appear without triggering the click?

Reply:
The `connectionsClass` string is [computed](https://github.com/Human-Connection/Human-Connection/pull/3293/files#diff-68c74572b7e55ffbcef954e9c4d9d487R77) and then used in [line 7](https://github.com/Human-Connection/Human-Connection/pull/3293/files#diff-68c74572b7e55ffbcef954e9c4d9d487R7)